### PR TITLE
Query TOR Network For IP Address and Domain names Search API

### PR DESCRIPTION
**Pull request by neetinkandhare**
## Motivation and Context

Fixes #

## Checklist:

- [] I have added a [Signed-off-by](https://github.com/IBMResilient/resilient-community-apps/blob/master/CONTRIBUTING.md)
- [] Either no new documentation is required by this change, OR I added new documentation
- [] Either no new tests are required by this change, OR I added new tests
- [] I have run pep8 and pylint. I have cleaned up all valid errors and warnings in code I have added or modified. These tools may generate false positives. Don't be worried about ignoring some errors or warnings. The goal is clean, consistent, and readable code.

Signed-off-by:

### CODE REVIEW

File: fn_query_tar_network/fn_query_tar_network/components/query_tor_network.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            # PUT YOUR FUNCTION IMPLEMENTATION CODE HERE
+            yield StatusMessage("starting...")
+
+            __params_data = {'flag': self.options.get('flag'), 'fields': self.options.get('data_fields'),
+                             'search': search_data}
+
+            __response_data = requests.get(self.options.get('base_url'), params=__params_data)
+
+            __response_data = __response_data.text
+            if __response_data.find(search_data) != -1:
```
Comment (mscherfling) [inline fn_query_tar_network/fn_query_tar_network/components/query_tor_network.py]: My first impression is that using find could cause false positive searches. If you can review and let me know that you're confident that false positives are not possible. For instance, what if the artifact value was 'or_addr' or 'Valid'?  Each of these are in the response but meaning data to search for. I know these are unlikely artifact values, but this issue should be considered.

File: fn_query_tar_network/fn_query_tar_network/util/config.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+
+def config_section_data():
+    """Produce the default configuration section for app.config,
+       when called by `resilient-circuits config [-c|-u]`
+    """
+    config_data = u"""[fn_query_tar_network]
+base_url = https://onionoo.torproject.org/details
+flag = Exit  
+data_fields = exit_addresses,or_addresses,host_name
```
Comment (mscherfling) [inline fn_query_tar_network/fn_query_tar_network/util/config.py]: add a comment to indicate that a comma separated format should be used. Did you test if someone code extra spaces:
exit_address, or_addresses, host_name

don't know if that still works or you have to strip out blanks

File: fn_query_tar_network/tests/test_query_tor_network.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
+# (c) Copyright IBM Corp. 2010, 2018. All Rights Reserved.
```
Comment (mscherfling) [inline fn_query_tar_network/tests/test_query_tor_network.py]: let's remove this file is it's not used.

Comment (neetinkandhare) [reply fn_query_tar_network/tests/test_query_tor_network.py]: Raising New code Request hence closing the PR

**Comment by neetinkandhare** [comment]
Raising the New Pull Request with Modified Code, have incorporated all above review comments except that 'Find' keyword related.  i will make discover with this keyword and let you know.

**Pull request closed without merging**